Guesser/FilterTypeGuesser.php

```diff
@@ -77,18 +85,21 @@ public function guessType($class, $property, ModelManagerInterface $modelManager
                 return new TypeGuess('doctrine_orm_date', $options, Guess::HIGH_CONFIDENCE);
             case 'decimal':
             case 'float':
-                $options['field_type'] = 'number';
-
-                return new TypeGuess('doctrine_orm_number', $options, Guess::MEDIUM_CONFIDENCE);
```

Comment:
Another PR would be maybe a bit overkill, a separate commit is enough in this case IMO.